playbooks/templates/.env.j2

```diff
@@ -131,6 +131,14 @@ SKYNET_DASHBOARD_URL=https://account.{{ cluster_domain }}
 # Accounts log level
 SKYNET_ACCOUNTS_LOG_LEVEL=info
 
+{% if webportal_common_config.accounts_limit_access is defined %}
+# Accounts limit access to authenticated users
+# Possible values:
+# - 'authenticated': Only authenticated users can access the portal (default if unset or empty)
+# - 'false': All users incl. anonymous users can access the portal (access to portal is public)
+ACCOUNTS_LIMIT_ACCESS={{ webportal_common_config.accounts_limit_access }}
```

Comment:
if the intention is for `authenticated` to be the default, shouldn't we just set it like this:
```
ACCOUNTS_LIMIT_ACCESS={{ webportal_common_config.accounts_limit_access | default('authenticated') }}
```
and leave out the if?

Or is the default behavior of ACCOUNTS_LIMIT_ACCESS being defined somewhere else?
If it is being defined somewhere else, I would extend the comment to say that X system handle the default behavior setting so ansible should not handle the default case, and on needs to respond to if the flag is set or not.

Reply:
Default value if empty or unset is handled by `skynet-webportal/docker-compose.accounts.yml`.
The comment was updated.